config/settings/local.py

```diff
@@ -50,3 +50,28 @@
 CELERY_ALWAYS_EAGER = False
 # Your stuff...
 # ------------------------------------------------------------------------------
+
+CORS_ORIGIN_WHITELIST = [
```

Comment:
These should be based on environment variables, not hardcoded - shouls probably also be in base instead of local